src/popup/components/Protocol/EnsureUserSavedMnemonic.tsx

```diff
@@ -64,15 +64,11 @@ export function EnsureUserSavedMnemonic(props: { onComplete: () => void }) {
       const mnemonic = await getProtocolOptIn().getMnemonic();
       setMnemonicArr((mnemonic as string).split(" "));
       const sortedMnemonic = (mnemonic as string).split(" ").sort();
-      const tempButtons = [];
-      for (const word of sortedMnemonic) {
-        tempButtons.push({
-          word,
-          isEnabled: true,
-        });
-      }
-
-      setButtons(tempButtons);
+      setButtons(
+        sortedMnemonic.map((w) => {
+          return { word: w, isEnabled: true };
+        }),
```

Comment:
If you'd like: `sortedMnemonic.map(w => ({ word: w, isEnabled: true }))`. You can wrap the braces in parens to not have to use return like this.